sources/web/datalab/static/page.js

```diff
@@ -0,0 +1,231 @@
+function placeHolder() {}
+
+function reportEvent(event) {
+  var reportingEnabled = (document.body.getAttribute('data-reporting-enabled') == 'true');
+  if (!reportingEnabled) { return; }
+
+  var signedIn = (document.body.getAttribute('data-signed-in') == 'true');
+  var additionalMetadata = 'signedIn=' + signedIn;
+  if (event['metadata']) {
+    event['metadata'] = event['metadata'] + ',' + additionalMetadata;
+  } else {
+    event['metadata'] = additionalMetadata;
+  }
+  window.dataLayer = window.dataLayer || [];
+  window.dataLayer.push(event);
+}
+
+function toggleSidebar() {
```

Comment:
I renamed page.js to appbar.js and moved some common functions back into datalab.js.

As discussed offline, I will leave toggleSidebar and showHelp here for now, until we can figure out a cleaner detangling of appbar and notebook-app.